espnet2/bin/whisper_export_vocabulary.py

```diff
@@ -54,9 +56,14 @@ def export_vocabulary(output: str, whisper_model: str, log_level: str):
     # NOTE (Shih-Lun): extra tokens (for timestamped ASR) not
     #                  stored in the wrapped tokenizer
     full_vocab_size = 51865 if whisper_model == "whisper_multilingual" else 51864
+
     for i in range(full_vocab_size - vocab_size):
         fout.write("()" + "\n")
 
+    if sot_asr:
+        full_vocab_size += 1
+        fout.write("<sc>" + "\n")
```

Comment:
Is it possible to import custom tokens from a file, like additional_vocab. This would allow us to add any tokens as needed, rather than only adding a <sc> token for the SOT training.

Reply:
The Whisper's vocabulary is not imported form a file, but loaded form the whisper pypi package. So, here I add the special token in the code.

Reply:
I think it would be better to have at least a way to specify which special token would be used as a speaker change. The hard-corded token symbol may confuse the user.

I guess this would depend on other places, so if it is difficult to make this speaker change token a variable, we could at least provide the comment in the source code and README that <sc> is a reserved token in an appropriate place.

Reply:
Thanks for your suggestions! Now it's configurable.